Runtime/Scripts/MeshSyncServer.Properties.cs

```diff
@@ -0,0 +1,306 @@
+using System.Collections.Generic;
+using UnityEngine;
+#if AT_USE_SPLINES
+using UnityEngine.Splines;
+using Unity.Mathematics;
+#endif
+
+#if AT_USE_PROBUILDER && UNITY_EDITOR
+using UnityEngine.ProBuilder;
+#endif
+
+namespace Unity.MeshSync
+{
+    partial class MeshSyncServer
+    {
+        public enum PropertiesState
+        {
+            None,
+            Sending,
+            Received
+        }
+
+        bool needsClientSync;
+
+        Dictionary<EntityRecord, Matrix4x4> entityTransforms = new Dictionary<EntityRecord, Matrix4x4>();
+
+#if UNITY_EDITOR
+        public override InstanceHandlingType InstanceHandling
```

Comment:
Failed to notice this before. Why do we need to make this public ?
And if so, we would need the API documentation too.

Reply:
I made it internal now.